ios/ReactNativePageView.m

```diff
@@ -1,11 +1,216 @@
 
 #import "ReactNativePageView.h"
 #import "React/RCTLog.h"
+#import <React/RCTViewManager.h>
+
+
+@interface RCTOnPageScrollEvent : NSObject <RCTEvent>
+
+- (instancetype) initWithReactTag:(NSNumber *)reactTag
+                         position:(NSNumber *)position
+                           offset:(NSNumber *)offset
+                    coalescingKey:(uint16_t)coalescingKey;
+
+@end
+
+@implementation RCTOnPageScrollEvent
+{
+    NSNumber* _position;
+    NSNumber* _offset;
+    uint16_t _coalescingKey;
```

Comment:
Do we really need this coalescingKey?

Reply:
Yes, it's not obvious for be why, but we have to implement it always

Reply:
Nvm, I checked it on `react-native` master branch :) It is optional from version `0.60` Sorry :D